src/Orc.Controls/Controls/NumericUpDown/Converters/ByteToDecimalValueConverter.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace Orc.Controls.Converters
+{
+    using System;
+    using Catel.MVVM.Converters;
+
+    public class ByteToDecimalValueConverter : ValueConverterBase<byte, decimal>
+    {
+        #region Methods
+        protected override object Convert(byte value, Type targetType, object parameter)
+        {
+            return System.Convert.ToDecimal(value);
+        }
+
+        protected override object ConvertBack(decimal value, Type targetType, object parameter)
+        {
+            return System.Convert.ToByte(value);
```

Comment:
Potential overflow exception
According to msdn:
`The data binding engine does not catch exceptions that are thrown by a user-supplied converter. Any exception that is thrown by the Convert method, or any uncaught exceptions that are thrown by methods that the Convert method calls, are treated as run-time errors. Handle anticipated problems by returning DependencyProperty.UnsetValue.`
You can also return validation result (i think in this cases it will pass the error to wpf validation, but not sure)

Reply:
yes, you're right for all converters

Reply:
actually this Numeric up down was by design Decimal control and we assume that Decimal type is sufficient for 99% cases... This isn't right actually...and i don't like much this whole converters thing (converting double-> decimal/ decimal -> double don't feel like a right thing)....  maybe it would be better to create separate UpDown control for each numeric type (think it's too much for now)... so the easiest way was to create 1 control and converters for Each numeric type To/From Double  ... there won't be 100% right way to do so(... at least i can provide protection for type casting process you mention...

Reply:
hm... found this mistake! 
![image](https://user-images.githubusercontent.com/5163834/134602904-6973f25c-eb4b-4b7f-a241-e138c36d1675.png)

Reply:
should be decimal